schema.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const notes = mongoose.Schema({
+  category: { type: String, default: 'general', required:
+  true },
+  text: { type: String, required: true },
+});
+
+module.exports = mongoose.model('notes', notes);
```

Comment:
A note on cleaning this file structure a little bit, this can be moved into `lib/model/notes-schema.js`